models/public/forward_tacotron/description/forward_tacotron.md

```diff
@@ -0,0 +1,123 @@
+# ForwardTacotron
+
+## Use Case and High-Level Description
+
+ForwardTacotron is model for text to speech task originally trained on PyTorch\*
+then converted to ONNX\* format. Forward Tacotron is the model for mel-spectrogramm regression from text.
+For details see [paper](https://arxiv.org/pdf/1703.10135.pdf),[paper](https://arxiv.org/pdf/1905.09263.pdf), [repository](https://github.com/as-ideas/ForwardTacotron).
+
+### Steps to Reproduce PyTorch to ONNX Conversion
+Model is provided in ONNX format, which was obtained by the following steps.
+
+1. Clone the original repository
+```sh
+git clone https://github.com/as-ideas/ForwardTacotron
+cd ForwardTacotron
+```
+2. Checkout the commit that the conversion was tested on:
+```sh
+git checkout 78789c1aa845057bb2f799e702b1be76bf7defd0
+```
+3. Follow README.md and train ForwardTacotron model.
+4. Copy provided scripts `forward_to_onnx.py` to ForwardTacotron root directory.
+
+5. Create pretrained directory and copy the best model to it.
+```sh
+mkdir pretrained
+cp checkouts/ljspeech_tts.forward/forward_<iteration>k.pyt pretrained
+```
```

Comment:
Why is this step necessary? Your conversion script can read the model from the original path.

Also, ITYM `checkpoints`, not `checkouts`.